### DISCUSSION TITLE
ESP32 2432S028 Nerdminer Display values

### DISCUSSION THREAD

**Discussion by micha-uhl** [BitMaker-hub/NerdMiner_v2#393] [General]
On the display, the values for 32bit shares, workers and total hashrate are always displayed with zero

**Comment by IAmSixNine** [comment]
If you connect to a pool other than public-pool the workers will always show 0. But i am pretty sure my 32 bit value has changed over time when connected to a different pool. I think.

**Comment by micha-uhl** [comment]
thank you, with public-pool it works!

**Reply by bellven** [reply]
What do you do to make it work?

**Reply by m8ax** [reply]
probe my firmware https://m8ax.github.io/MvIiIaX-NerdMiner_V2-DeV/
